src/redirect.js

```diff
@@ -0,0 +1,20 @@
+import { STILLINGSOK_URL } from './fasitProperties';
+
+const whiteList = [
+    '/pam-stillingsok/favoritter',
```

Comment:
Er det bedre å bruke {CONTEXT_PATH} her i stedet for å hardkkode /pam-stillingsok/?

Reply:
Ja det er det. Fikser det :)